crazyflie-link/src/packet.rs

```diff
@@ -0,0 +1,102 @@
+use std::fmt;
+
+pub struct Packet {
+    port: u8,
+    channel: u8,
+    data: Vec<u8>,
+}
+
+impl fmt::Display for Packet {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(
+            f,
+            "(ch: {}, port: {}, data: {})",
+            self.channel, self.port, self.data[0]
+        )
+    }
+}
+
+impl From<Vec<u8>> for Packet {
+    fn from(v: Vec<u8>) -> Self {
+        let header = v[0];
+        //
+        // A packet on the wire starts with a header byte, formatted as:
+        //
+        //   pppp11cc
+        //
+        // Where ...
+        //  ... bit 1 and 2 is the channel (c) ...
+        //  ... bit 3 and 4 is legacy and set to 1 ...
```

Comment:
Bit 3-4 are "Link-reserved". For the radio link they are used for flow control, on the other links they are unused. If set to "11", it breaks the benchmark example (see above).

The current implementation was [setting both reserved bit to 0](https://github.com/ataffanel/crazyflie-link-rs/blob/7d2ead8415b0e549315dc2d165a474f020b2d435/crazyflie-link/src/connection.rs#L219) when receiving a packer, we should likely keep them at 0.